RELEASE_NOTES.md

```diff
@@ -0,0 +1,27 @@
+<!--
```

Comment:
Is the idea that we'd copy/paste this into a release description, or does GH handle the template automatically?

Reply:
Once we add goreleaser to this repo, this file will act as input. See the release process on Gaia for more details https://github.com/cosmos/gaia/blob/main/RELEASE_PROCESS.md#release-notes